ui/redux/reducers/comments.js

```diff
@@ -725,6 +733,21 @@ export default handleActions(
       };
     },
 
+    [ACTIONS.MEMBERS_ONLY_CHAT_TOGGLED]: (state: CommentsState, action: any) => {
+      l('action');
+      l(action)
+      return state;
+      // const { membersOnlyChatEnabled } = action.data;
+      // const commentById = Object.assign({}, state.commentById);
+      //
+      // if (!commentById[comment_id]) {
+      //   return state;
+      // }
+      //
+      // commentById[comment_id] = { ...commentById[comment_id], removed: true };
+      // return { ...state, commentById };
+    },
```

Comment:
:white_flag: Backwards software lifecycle

Non functional things like this shouldn't be happening at this stage of development, and definitely not if the branch is co-worked now (branch off instead).  

<img width="422" alt="image" src="https://user-images.githubusercontent.com/64950861/192926807-1f75f92c-2d5e-407a-bc43-13c8006a702c.png">